src/scala/scripts/PBGenerateRequest.scala

```diff
@@ -6,6 +6,25 @@ case class PBGenerateRequest(jarOutput: String, scalaPBOutput: Path, scalaPBArgs
 
 object PBGenerateRequest {
 
+  // This little function fixes a problem, where external/com_google_protobuf is not found. The com_google_protobuf
+  // is special in a way that it also brings-in protoc and also google well-known proto files. This, possibly,
+  // confuses Bazel and external/com_google_protobuf is not made available for target builds. Actual causes are unknown
+  // and this fixTransitiveProtoPath fixes this problem in the following way:
+  //  (1) We have a list of all required .proto files; this is a tuple list (root -> full path), for example:
+  //        bazel-out/k8-fastbuild/bin -> bazel-out/k8-fastbuild/bin/external/com_google_protobuf/google/protobuf/source_context.proto
+  //  (2) Convert the full path to relative from the root:
+  //        bazel-out/k8-fastbuild/bin -> external/com_google_protobuf/google/protobuf/source_context.proto
+  //  (3) From all the included protos we find the first one that is located within dir we are processing -- relative
+  //      path starts with the dir we are processing
+  //  (4) If found -- the include folder is "orphan" and is not anchored in either host or target. To fix we prepend
+  //      root. If not found, return original. This works as long as "external/com_google_protobuf" is available in
+  //      target root.
+  def fixTransitiveProtoPath(includedProto: List[(Path, Path)])(orig: String): String = includedProto
```

Comment:
nit: can we store the first map through the list:
```scala
def fixTransitiveProtoPath(includedProto: List[(Path, Path)]): String => String = {
  val relPath = includedProto.map { case (root, full) => (root.toString, root.relativize(full).toString) }

  { orig: String =>
    relPath.find { case (_, rel) => rel.startsWith(orig) } match {
       case Some((root, _)) => s"$root/$orig"
       case None => orig
    }
  } 
}
```
This is a bit longer, but it builds the list and converts to strings once.

Reply:
Good point! Optimized.